feature/fortune/src/main/java/org/sopt/official/feature/fortune/feature/fortuneDetail/component/FortuneDetailBox.kt

```diff
@@ -0,0 +1,43 @@
+package org.sopt.official.feature.fortune.feature.fortuneDetail.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.official.designsystem.Gray700
+
+@Composable
+internal fun TodayFortuneBox(
+    content: @Composable () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        contentAlignment = Alignment.Center,
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(horizontal = 20.dp)
+            .background(
+                color = Gray700,
+                shape = RoundedCornerShape(12.dp),
+            ),
+    ) {
+        content()
+    }
+}
```

Comment:
이부분은 분리한 이유가 있나요??
배경 색상과 모양을 나타내고 있기 때문에 따로 분리 후 Slot을 뚫어서 써준.. 그런 느낌인걸까요?

Reply:
카드 형태의 뷰면 차라리 

```suggestion
@Composable
internal fun TodayFortuneCard(
    content: @Composable () -> Unit,
    modifier: Modifier = Modifier,
) {
    Box(
        contentAlignment = Alignment.Center,
        modifier = modifier
            .fillMaxWidth()
            .padding(horizontal = 20.dp)
            .background(
                color = Gray700,
                shape = RoundedCornerShape(12.dp),
            ),
    ) {
        content()
    }
}
```
로 하는게 좀 더 직관적을 것 같다는 생각이 들어요

Reply:
fortuneDetail 패키지 내부에서 사용할 공용 컴포넌트로 솝마디 대시보드와 콕 찌르기 대시보드의 배경 Box 컴포넌트입니다.
개인적으로 패키지 내부에서 분리하는 기준은 1. 테스트 2. 프리뷰 3. 뷰 내의 공용 컴포넌트입니다.
Card보단 Box가 어울리지 않나용?

Reply:
ㅇㅎ 하긴 border도 없고 그냥 RoundedCornerShape만 있네요. 이대로 가도 좋을 것 같습니다